osgp/platform/osgp-throttling-service/src/main/java/org/opensmartgridplatform/throttling/services/RedisPermitService.java

```diff
@@ -0,0 +1,171 @@
+// SPDX-FileCopyrightText: Copyright Contributors to the GXF project
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.opensmartgridplatform.throttling.services;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.StreamSupport;
+import org.opensmartgridplatform.throttling.model.NetworkSegment;
+import org.opensmartgridplatform.throttling.model.Permit;
+import org.opensmartgridplatform.throttling.model.PermitKey;
+import org.redisson.api.RKeys;
+import org.redisson.api.RLock;
+import org.redisson.api.RScoredSortedSet;
+import org.redisson.api.RedissonClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RedisPermitService implements PermitService {
+
+  private static final Logger log = LoggerFactory.getLogger(RedisPermitService.class);
+
+  private static final int TRY_LOCK_TIME_MS = 100;
+  private static final int WAIT_TIME_MS = 1000;
```

Comment:
Is het nog iets om TRY_LOCK_TIME_MS en WAIT_TIME_MS ook configurabel te maken? Ik weet niet op basis waarvan deze waarden gekozen zijn.

Reply:
Het is een korte pauze na het verkrijgen van een lock en voordat daadwerkelijk geprobeerd wordt om een permit te verkrijgen. De seconde komt uit de bestaande implementatie, die heb ik overgenomen. Deze moeten niet configurabel te zijn.